src/sql/workbench/browser/modal/modal.ts

```diff
@@ -412,6 +412,24 @@ export abstract class Modal extends Disposable implements IThemable {
 		});
 	}
 
+	/**
+	* removes the footer button matching the provided label
+	* @param label Label on the button
+	*/
+	protected removeFooterButton(label: string): void {
+		let buttonIndex = this._footerButtons.findIndex(e => {
+			try {
```

Comment:
Curious why the try/catch is necessary - I wouldn't expect this to every throw.

Reply:
copied the same patter as find... e.element might throw in worst case

Reply:
Why not just add to the check below if you're concerned about e.element being undefined?

```
return e && e.element && e.element.innerText
```

It just seems unnecessarily verbose.